charts/air-discount-scheme/values.dev.yaml

```diff
@@ -6,6 +6,7 @@ air-discount-scheme-api:
     CONTENTFUL_HOST: preview.contentful.com
     ELASTIC_NODE: https://vpc-search-njkekqydiegezhr4vqpkfnw5la.eu-west-1.es.amazonaws.com
     SAML_ENTRY_POINT: https://innskraning.island.is/?id=ads.dev
+    SSF_ON: ''
```

Comment:
we could possibly do with a longer but more descriptive name

Reply:
Agreed, will fix

Reply:
fixed